docs-ref-conceptual/dotnet-tools.md

```diff
@@ -72,18 +72,46 @@ During install, Azure tools are already enabled by default.
 
 ## [Linux](#tab/linux)
 
-<img src="https://docs.microsoft.com/en-us/visualstudio/products/images/vs-code.svg" width="50" height="50"></img>
+<table>
+  <tr>
+    <td width="50">
+        <img src="https://docs.microsoft.com/en-us/visualstudio/products/images/vs-code.svg" width="50" height="50"></img>
+    </td>
+    <td>
+        Visual Studio Code has everything you need for Azure development on Linux. In addition, Visual Studio supports ASP.NET Core apps on Azure App Service for Linux.
+    </td>
+  </tr>
+</table>
 
-### Step 1: Download the .NET Core SDK
+### Step 1: Download the .NET Core SDK (VS Code)
 
 The SDK and command-line tools for .NET Core apps.
 
 > [!div class="nextstepaction"]
 > [Download .NET Core SDK](https://www.microsoft.com/net/core)
 
-### Step 2: Visual Studio Code
+### Step 2: Visual Studio Code (VS Code)
 
 Edit and debug .NET Core apps on any operating system: Windows, Mac, and Linux.
 
 > [!div class="nextstepaction"]
 > [Download Visual Studio Code](https://code.visualstudio.com)
+
+For development with Visual Studio, take the following steps instead.
```

Comment:
I understand why you made these changes, but this page is intended to list tools for .NET development **on** Linux, not **for** Linux. Visual Studio 2017 doesn't run on Linux.  Well, not *yet*, anyway.  😄